frontend/zc_messaging/src/pages/threads/index.jsx

```diff
@@ -1,13 +1,65 @@
+import { useEffect, useState } from "react"
 import { Helmet } from "react-helmet"
+import { useSelector } from "react-redux"
+
+import "./threads.css"
+import { ThreadList } from "../../component"
+
 import generatePageTitle from "../../utils/generatePageTitle"
+import { getThreadHandler } from "./threads.utils"
 
 const Threads = () => {
+  const authUser = useSelector(state => state.authUser)
+  const [threads, setThreads] = useState([])
+  const jasonaParseData = data => {
+    return JSON.parse(data)
+  }
+
+  useEffect(() => {
+    const userInfo = localStorage.getItem("userData")
+    const new_data = jasonaParseData(userInfo)
+    const org_ud = new_data.currentWorkspace
+    const sender_ud = new_data.user._id
+
+    const threads_data = getThreadHandler(org_ud, sender_ud)
+    if (threads_data != []) {
+      threads_data.then(respsonse => {
+        let threads_array = respsonse.data
+        let dumpy_threads_Arr = []
+        for (let i = 0; i < threads_array.length; i++) {
+          for (let j = 0; j < threads_array[i].threads.length; j++) {
+            dumpy_threads_Arr.push(threads_array[i].threads[j])
+          }
+        }
+        setThreads(dumpy_threads_Arr)
+      })
+    } else {
+      console.warn("Nothing was returned")
+    }
+  }, [])
+
+  const getThreadFunc = (orgId, senderId) => {
+    console.warn(authUser)
```

Comment:
same here

Reply:
I'll take it out. console.log was disabled on eslint that's why I was using .warn to read out errors.